src/scripts/h5p-multi-media-choice-option.js

```diff
@@ -249,6 +250,27 @@ export class MultiMediaChoiceOption {
     this.wrapper.appendChild(this.accessibilitySolutionText);
   }
 
+  /**
+   * Shows feedback below answer (if any is available)
+   */
+  showFeedback() {
+    if (!this.feedback) {
+      return;
+    }
+    if (this.isSelected() && this.feedback.chosenFeedback) {
+      this.addFeedbackText(this.feedback.chosenFeedback);
+    }
+    else if (!this.isSelected() && this.feedback.notChosenFeedback) {
+      this.addFeedbackText(this.feedback.notChosenFeedback);
+    }
+  }
+
+  addFeedbackText(html) {
+    this.feedbackText = document.createElement('div');
```

Comment:
It's common practice in H5P content types to assign a class to every element. It makes CSS customization easier.